client-js/app/elements/labrad-plot.ts

```diff
@@ -1,9 +1,25 @@
 import 'd3';
-
+import {THREE} from 'three';
```

Comment:
I get runtime errors from this. I think this needs to get the _default_ import, rather than an item named 'THREE' from inside the module. That is:
```typescript
import THREE from 'three';
```